util-logging/src/test/scala/com/twitter/logging/FileHandlerTest.scala

```diff
@@ -302,5 +302,49 @@ class FileHandlerTest extends WordSpec with TempFolder {
         handler.close()
       }
     }
+
+    /**
+      * Execute this test individually to affirm that the roll policy based upon the size works
+      * even after the test case is executed couple of times.
+      * This test mimics the scenario that, if log file exists, FileHandler should pick up it's size
```

Comment:
I don't love this testing style.  how about instead having two Filehandlers in one test?